lib/mediators/followups/notification_updater.rb

```diff
@@ -45,7 +45,7 @@ def create_notifications_for_all_collabs
     end
 
     def current_collab_hids
-      current_collabs.map {|c| c.user.heroku_id }
+      @current_collabs ||= current_collabs.map {|c| c.user.heroku_id }
```

Comment:
```suggestion
      @current_collab_hids ||= current_collabs.map {|c| c.user.heroku_id }
```

This may be pedantic, but there is another method named `current_collabs`, if that were ever memoized, we could see collisions.

Reply:
@drapergeek not pedantic at all, because I mixed them up! Thank goodness for peer review!